SPECS/aide/aide.spec

```diff
@@ -0,0 +1,321 @@
+Summary:        Intrusion detection environment
+Name:           aide
+Version:        0.16
+Release:        15%{?dist}
+License:        GPLv2+
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://sourceforge.net/projects/aide
+Source0:        https://github.com/aide/aide/releases/download/v%{version}/%{name}-%{version}.tar.gz
+Source1:        aide.conf
+Source2:        README.quickstart
+Source3:        aide.logrotate
+# Customize the database file location in the man page.
+Patch1:         aide-0.16rc1-man.patch
+# fix aide in FIPS mode
+Patch2:         aide-0.16b1-fipsfix.patch
+# Bug 1674637 - aide: FTBFS in Fedora rawhide/f30
+Patch3:         aide-0.16-Use-LDADD-for-adding-curl-library-to-the-linker-comm.patch
+Patch4:         aide-0.15-syslog-format.patch
+Patch5:         aide-0.16-crypto-disable-haval-and-others.patch
+Patch6:         coverity.patch
+Patch7:         aide-0.16-crash-elf.patch
+BuildRequires:  audit-devel
+BuildRequires:  autoconf
+BuildRequires:  automake
+BuildRequires:  bison
+BuildRequires:  curl-devel
+BuildRequires:  e2fsprogs-devel
+BuildRequires:  flex
+BuildRequires:  gcc
+BuildRequires:  libacl-devel
+BuildRequires:  libattr-devel
+BuildRequires:  libgcrypt-devel
+BuildRequires:  libgpg-error-devel
+BuildRequires:  libtool
+BuildRequires:  make
+BuildRequires:  pcre-devel
+BuildRequires:  pkg-config
+BuildRequires:  zlib-devel
+BuildRequires:  pkgconfig(libselinux)
+
+%description
+AIDE (Advanced Intrusion Detection Environment) is a file integrity
+checker and intrusion detection program.
+
+%prep
+%autosetup -p1
+cp -a %{SOURCE2} .
+
+%build
+autoreconf -ivf
+%configure  \
+  --disable-static \
+  --with-config_file=%{_sysconfdir}/aide.conf \
+  --with-gcrypt \
+  --with-zlib \
+  --with-curl \
+  --with-posix-acl \
+  --with-selinux \
+  --with-xattr \
+  --with-e2fsattrs \
+  --with-audit
+%make_build
+
+%install
+%make_install bindir=%{_sbindir}
+install -Dpm0644 -t %{buildroot}%{_sysconfdir} %{SOURCE1}
+install -Dpm0644 %{SOURCE3} %{buildroot}%{_sysconfdir}/logrotate.d/aide
+mkdir -p %{buildroot}%{_localstatedir}/log/aide
+mkdir -p -m0700 %{buildroot}%{_localstatedir}/lib/aide
+
+%files
+%license COPYING
+%doc AUTHORS ChangeLog NEWS README doc/manual.html contrib/
+%doc README.quickstart
+%{_sbindir}/aide
+%{_mandir}/man1/*.1*
+%{_mandir}/man5/*.5*
+%config(noreplace) %attr(0600,root,root) %{_sysconfdir}/aide.conf
+%config(noreplace) %{_sysconfdir}/logrotate.d/aide
+%dir %attr(0700,root,root) %{_localstatedir}/lib/aide
+%dir %attr(0700,root,root) %{_localstatedir}/log/aide
+
+%changelog
+* Fri Aug 06 2021 Bala <balakumaran.kannan@microsoft.com> - 0.16-15
+- Initial CBL-Mariner import from Fedora 32 (license: MIT)
```

Comment:
Please check the package's license and once verified (and potentially updated the `License` tag) indicate that with `License verified` in the changelog.